### PR TITLE
Add link to PrivateBin ansible role

### PR DESCRIPTION
**Pull request by e1mo**
This role simply deploys PrivateBin on your servers, it's functionality
is quite limited and currently does not support further hardening
techniques[^0]. But however, I think it is beneficial for quite a lot of
people.

[^0]: https://todo.sr.ht/~e1mo/ansible-role-privatebin/1

## ToDo

* [ ] Feedback on the role

### CODE REVIEW

**Review by rugk — APPROVED** [review]

**Comment by rugk** [comment]
Hi @e1mo,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

What do you mean by „ Feedback on the role“? Should we as PrivateBin devs give feedback here?

File: INSTALL.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ options](#configuration) to adjust as you see fit.
 
 **NOTE:** See [our FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ#how-can-i-securely-clonedownload-your-project) for information how to securely download the PrivateBin release files.
 
+**NOTE:** There is a [ansible](https://ansible.com) role by @e1mo available to install and configure PrivateBin on your server. It's available on [ansible galaxy](https://galaxy.ansible.com/e1mo/privatebin) ([Source code](https://git.sr.ht/~e1mo/ansible-role-privatebin)).
```
Comment (rugk) [inline INSTALL.md:12]:
```suggestion
**NOTE:** There is a [ansible](https://ansible.com) role by @e1mo available to install and configure PrivateBin on your server. It's available on [ansible galaxy](https://galaxy.ansible.com/e1mo/privatebin) ([source code](https://git.sr.ht/~e1mo/ansible-role-privatebin)).

Maybe like that (that's IMHO better).
```

**Comment by e1mo** [comment]
Thanks for your quick response!

> What do you mean by „ Feedback on the role“? Should we as PrivateBin devs give feedback here?

It is more a general invitation to everyone who uses / knows ansible to look at the role and share ideas for improvement. It's no merge-blocking TODO.

**Comment by elrido** [comment]
Sorry, I missed this.

The role is functional and already pretty flexible with the configuration file generation. It leaves the installation of a webserver and php as an excercise to the user, but that makes it compatible with a broad range of other roles and different distros and web server setups.

As was mentioned, it doesn't currently apply the hardening recommendations. If you'd like to look into how that could be done in an automated fashion, here is the relevant section of the docker container build (written for busybox shell, but should be straight forward to replicate using common ansible modules):
https://github.com/PrivateBin/docker-nginx-fpm-alpine/blob/7dfe1b2bb4a11ac1d6732a94cea50517eb58864f/Dockerfile#L21-L35

For reference to other ansible users, this is a snippet of the ansible tasks used to setup the demo instance on a host with docker:
```
- name: PrivateBin demo persistent volume
  file:
    path: /srv/privatebin-demo
    state: directory
    owner: "{{ (docker_userns_offset + 65534) | string }}"
    group: "{{ (docker_userns_offset + 82) | string }}"
    mode: 0750

- name: PrivateBin demo configuration
  copy:
    src: conf.php
    dest: /srv/privatebin-demo-conf.php
    owner: "{{ (docker_userns_offset + 65534) | string }}"
    group: "{{ (docker_userns_offset + 82) | string }}"
    mode: 0640

- name: create IPv6 network
  docker_network:
    name: ipv6
    enable_ipv6: yes
    ipam_config:
      - subnet: "{{ docker_privatebin_ipv6_prefix }}/80"

- name: PrivateBin demo container
  docker_container:
    name: privatebin-demo
    image: privatebin/nginx-fpm-alpine:1.3.4-alpine3.12.1
    state: started
    restart_policy: always
    privileged: no
    read_only: yes
    init: yes
    networks_cli_compatible: no
    networks:
      - name: ipv6
    env:
      TZ: Europe/Zurich
      PHP_TZ: Europe/Zurich
    volumes:
      - /srv/privatebin-demo:/srv/data:rw
      - /srv/privatebin-demo-conf.php:/srv/cfg/conf.php:ro
```

**Comment by e1mo** [comment]
Hi, thanks for the dockerfile-hint! I'll be definitely taking a look at
it and try to incorporate those changes.

**Pull request merged**